--- ./settings.pro~	2005-09-08 04:01:29.000000000 +0300
+++ ./settings.pro	2005-09-08 16:21:34.000000000 +0300
@@ -21,12 +21,12 @@
 DEFINES += HAVE_MMX
 
 release {
-    QMAKE_CXXFLAGS_RELEASE = -O3 -march=pentiumpro -fomit-frame-pointer
+    QMAKE_CXXFLAGS_RELEASE += $${ECXXFLAGS}
     macx {
         # Don't use -O3, it causes some Qt moc methods to go missing
         QMAKE_CXXFLAGS_RELEASE = -O2
     }
-    QMAKE_CFLAGS_RELEASE = $${QMAKE_CXXFLAGS_RELEASE}
+    QMAKE_CFLAGS_RELEASE = $${ECFLAGS}
 }
 
 macx {
