diff -urN mythplugins-0.22rc1.orig/mythmusic/mythmusic/visualize.h mythplugins-0.22rc1/mythmusic/mythmusic/visualize.h
--- mythplugins-0.22rc1.orig/mythmusic/mythmusic/visualize.h	2009-08-08 02:36:08.000000000 +0200
+++ mythplugins-0.22rc1/mythmusic/mythmusic/visualize.h	2009-10-20 20:47:08.000000000 +0200
@@ -28,11 +28,11 @@
 #define myth_fftw_float double /* need to use different plan function to change */
 #define fftw_real myth_fftw_float
 #define myth_fftw_complex std::complex<myth_fftw_float>
-#if (myth_fftw_float == double)
+/*#if (myth_fftw_float == double)*/
 #define myth_fftw_complex_cast fftw_complex
-#elif (myth_fftw_float == float)
+/*#elif (myth_fftw_float == float)
 #define myth_fftw_complex_cast fftwf_complex
-#endif
+#endif*/
 #elif	FFTW2_SUPPORT
 #include <rfftw.h>
 #include <fftw.h>
