diff -ur ./mythweb/includes/init.php /tmp/mythweb/includes/init.php
--- ./mythweb/includes/init.php	2005-02-28 00:52:48.000000000 +0200
+++ /tmp/mythweb/includes/init.php	2005-10-26 00:11:25.000000000 +0300
@@ -14,7 +14,7 @@
     require_once 'includes/errordisplay.php';
 
 // Load the user-defined configuration settings
-    require_once 'config/conf.php';
+    require_once '/etc/mythweb/conf.php';
 
 // Clean up some variables
     if (!ereg('/$', $_SERVER['DOCUMENT_ROOT']))
--- ./mythweb/config/conf.php~	2005-11-04 02:25:34.000000000 +0200
+++ ./mythweb/config/conf.php	2005-11-04 02:26:32.000000000 +0200
@@ -29,7 +29,7 @@
 //    defined as mythtv.mydomain.com in server_domain above, mail will go to
 //    mythweb_errors@mydomain.com.
 //
-    define('error_email', 'mythweb_errors@'.preg_replace('/.*?\b([\w\-]+\.[\w\-]+)$/', '$1', server_domain));
+    define('error_email', 'root@'.preg_replace('/.*?\b([\w\-]+\.[\w\-]+)$/', '$1', server_domain));
 
 // For the "movies" search -- set this to the word your listings provider uses to
 //   describe movies/films/peliculas/etc.
--- /mythweb/includes/mythbackend.php~	2005-10-22 13:31:12.000000000 +0300
+++ /mythweb/includes/mythbackend.php	2005-11-04 23:31:31.707600269 +0200
@@ -335,7 +335,7 @@
             // Make sure the local path exists
                 $path = '';
                 foreach (split('/+', dirname($pngpath)) as $dir) {
-                    $path .= $path ? '/' . $dir : $dir;
+					$path .= $path ? ('/' . $dir) : ($dir ? $dir : "/$dir");
                     if(!is_dir($path) && !mkdir($path, 0755))
                         trigger_error('Error creating path for '.$path.': Please check permissions.', FATAL);
                 }
