--- mythplugins/mythmusic/mythmusic/cdrip.cpp.fix	2006-02-05 10:37:08.815231000 +0100
+++ mythplugins/mythmusic/mythmusic/cdrip.cpp	2006-02-05 10:37:43.505231000 +0100
@@ -9,8 +9,8 @@
 #include <fcntl.h>
 
 // Linux C includes
-#include <cdaudio.h>
 extern "C" {
+#include <cdaudio.h>
 #include <cdda_interface.h>
 #include <cdda_paranoia.h>
 }
--- mythplugins/mythmusic/mythmusic/main.cpp.fix	2006-02-05 10:46:03.285231000 +0100
+++ mythplugins/mythmusic/mythmusic/main.cpp	2006-02-05 10:48:00.065231000 +0100
@@ -9,9 +9,9 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <unistd.h>
-
+extern "C" {
 #include <cdaudio.h>
-
+}
 #include "decoder.h"
 #include "metadata.h"
 #include "maddecoder.h"
--- mythplugins/mythmusic/mythmusic/cddecoder.h.fix	2006-02-05 10:46:09.965231000 +0100
+++ mythplugins/mythmusic/mythmusic/cddecoder.h	2006-02-05 10:48:28.305231000 +0100
@@ -3,8 +3,8 @@
 
 #include "decoder.h"
 
-#include <cdaudio.h>
 extern "C" {
+#include <cdaudio.h>
 #include <cdda_interface.h>
 #include <cdda_paranoia.h>
 }
