--- ./configure~	2005-11-04 01:23:51.000000000 +0200
+++ ./configure	2005-11-05 16:18:39.000000000 +0200
@@ -250,7 +250,7 @@
         done
     fi
 
-    LIBPATHS="`echo $LIBPATHS $prefix/lib /lib /usr/lib /usr/local/lib /usr/X11R6/lib | sed s/'\/ '/' '/g` "
+    LIBPATHS="`echo $LIBPATHS $prefix/lib64 /lib64 /usr/lib64 /usr/local/lib64 /usr/X11R6/lib64 | sed s/'\/ '/' '/g` "
 
     HAS_IT="no"
     for LIBPATH in $LIBPATHS ; do
diff -u -r mythplugins-lib32/mythbrowser/mythbookmarkmanager/mythbookmarkmanager.pro mythplugins/mythbrowser/mythbookmarkmanager/mythbookmarkmanager.pro
--- mythplugins-lib32/mythbrowser/mythbookmarkmanager/mythbookmarkmanager.pro	2005-10-22 13:31:23.000000000 +0300
+++ mythplugins/mythbrowser/mythbookmarkmanager/mythbookmarkmanager.pro	2005-10-22 18:37:31.000000000 +0300
@@ -7,7 +7,7 @@
 TEMPLATE = lib
 CONFIG += plugin thread warn_on debug
 TARGET = mythbookmarkmanager
-target.path = $${PREFIX}/lib/mythtv/plugins
+target.path = $${PREFIX}/lib64/mythtv/plugins
 INSTALLS += target
 
 #uifiles.path = $${PREFIX}/share/mythtv/themes/default
diff -u -r mythplugins-lib32/mythcontrols/mythcontrols/mythcontrols.pro mythplugins/mythcontrols/mythcontrols/mythcontrols.pro
--- mythplugins-lib32/mythcontrols/mythcontrols/mythcontrols.pro	2005-10-22 13:31:35.000000000 +0300
+++ mythplugins/mythcontrols/mythcontrols/mythcontrols.pro	2005-10-22 18:37:31.000000000 +0300
@@ -4,7 +4,7 @@
 TEMPLATE = lib
 CONFIG += plugin thread
 TARGET = mythcontrols
-target.path = $${PREFIX}/lib/mythtv/plugins
+target.path = $${PREFIX}/lib64/mythtv/plugins
 INSTALLS += target
 
 uifiles.path = $${PREFIX}/share/mythtv/themes/default
diff -u -r mythplugins-lib32/mythdvd/mythdvd/mythdvd.pro mythplugins/mythdvd/mythdvd/mythdvd.pro
--- mythplugins-lib32/mythdvd/mythdvd/mythdvd.pro	2005-10-22 13:31:14.000000000 +0300
+++ mythplugins/mythdvd/mythdvd/mythdvd.pro	2005-10-22 18:37:31.000000000 +0300
@@ -9,7 +9,7 @@
 TEMPLATE = lib
 CONFIG += plugin thread
 TARGET = mythdvd
-target.path = $${PREFIX}/lib/mythtv/plugins
+target.path = $${PREFIX}/lib64/mythtv/plugins
 INSTALLS += target
 
 installfiles.path = $${PREFIX}/share/mythtv
diff -u -r mythplugins-lib32/mythgallery/mythgallery/mythgallery.pro mythplugins/mythgallery/mythgallery/mythgallery.pro
--- mythplugins-lib32/mythgallery/mythgallery/mythgallery.pro	2005-10-22 13:31:34.000000000 +0300
+++ mythplugins/mythgallery/mythgallery/mythgallery.pro	2005-10-22 18:37:31.000000000 +0300
@@ -10,7 +10,7 @@
 TEMPLATE = lib
 CONFIG += plugin thread
 TARGET = mythgallery
-target.path = $${PREFIX}/lib/mythtv/plugins
+target.path = $${PREFIX}/lib64/mythtv/plugins
 INSTALLS += target
 LIBS += -ltiff
 
diff -u -r mythplugins-lib32/mythgame/mythgame/mythgame.pro mythplugins/mythgame/mythgame/mythgame.pro
--- mythplugins-lib32/mythgame/mythgame/mythgame.pro	2005-10-22 13:31:44.000000000 +0300
+++ mythplugins/mythgame/mythgame/mythgame.pro	2005-10-22 18:37:31.000000000 +0300
@@ -4,7 +4,7 @@
 TEMPLATE = lib
 CONFIG += plugin thread
 TARGET = mythgame
-target.path = $${PREFIX}/lib/mythtv/plugins
+target.path = $${PREFIX}/lib64/mythtv/plugins
 INSTALLS += target
 
 installfiles.path = $${PREFIX}/share/mythtv
diff -u -r mythplugins-lib32/mythmusic/mythmusic/mythmusic.pro mythplugins/mythmusic/mythmusic/mythmusic.pro
--- mythplugins-lib32/mythmusic/mythmusic/mythmusic.pro	2005-10-22 13:31:22.000000000 +0300
+++ mythplugins/mythmusic/mythmusic/mythmusic.pro	2005-10-22 18:37:31.000000000 +0300
@@ -10,7 +10,7 @@
 TEMPLATE = lib
 CONFIG += plugin thread
 TARGET = mythmusic
-target.path = $${PREFIX}/lib/mythtv/plugins
+target.path = $${PREFIX}/lib64/mythtv/plugins
 INSTALLS += target
 
 installfiles.path = $${PREFIX}/share/mythtv
diff -u -r mythplugins-lib32/mythnews/mythnews/mythnews.pro mythplugins/mythnews/mythnews/mythnews.pro
--- mythplugins-lib32/mythnews/mythnews/mythnews.pro	2005-10-22 13:31:45.000000000 +0300
+++ mythplugins/mythnews/mythnews/mythnews.pro	2005-10-22 18:37:31.000000000 +0300
@@ -4,7 +4,7 @@
 TEMPLATE = lib
 CONFIG += plugin thread warn_on debug
 TARGET = mythnews
-target.path = $${PREFIX}/lib/mythtv/plugins
+target.path = $${PREFIX}/lib64/mythtv/plugins
 INSTALLS += target
 
 uifiles.path = $${PREFIX}/share/mythtv/themes/default
diff -u -r mythplugins-lib32/mythphone/mythphone/mythphone.pro mythplugins/mythphone/mythphone/mythphone.pro
--- mythplugins-lib32/mythphone/mythphone/mythphone.pro	2005-10-22 13:31:42.000000000 +0300
+++ mythplugins/mythphone/mythphone/mythphone.pro	2005-10-22 18:37:31.000000000 +0300
@@ -16,7 +16,7 @@
 TEMPLATE = lib
 CONFIG += plugin thread
 TARGET = mythphone
-target.path = $${PREFIX}/lib/mythtv/plugins
+target.path = $${PREFIX}/lib64/mythtv/plugins
 INSTALLS += target
 
 uifiles.path = $${PREFIX}/share/mythtv/themes/default
diff -u -r mythplugins-lib32/mythvideo/mythvideo/mythvideo.pro mythplugins/mythvideo/mythvideo/mythvideo.pro
--- mythplugins-lib32/mythvideo/mythvideo/mythvideo.pro	2005-10-22 13:31:39.000000000 +0300
+++ mythplugins/mythvideo/mythvideo/mythvideo.pro	2005-10-22 18:37:31.000000000 +0300
@@ -4,7 +4,7 @@
 TEMPLATE = lib
 CONFIG += plugin thread
 TARGET = mythvideo
-target.path = $${PREFIX}/lib/mythtv/plugins
+target.path = $${PREFIX}/lib64/mythtv/plugins
 INSTALLS += target
 
 uifiles.path = $${PREFIX}/share/mythtv/themes/default
diff -u -r mythplugins-lib32/mythweather/mythweather/mythweather.pro mythplugins/mythweather/mythweather/mythweather.pro
--- mythplugins-lib32/mythweather/mythweather/mythweather.pro	2005-10-22 13:31:33.000000000 +0300
+++ mythplugins/mythweather/mythweather/mythweather.pro	2005-10-22 18:37:31.000000000 +0300
@@ -4,7 +4,7 @@
 TEMPLATE = lib
 CONFIG += plugin thread
 TARGET = mythweather
-target.path = $${PREFIX}/lib/mythtv/plugins
+target.path = $${PREFIX}/lib64/mythtv/plugins
 INSTALLS += target
 
 uifiles.path = $${PREFIX}/share/mythtv/themes/default
--- ./mythplugins.libdir/mythflix/mythflix/mythflix.pro	2005-12-21 00:52:42.000000000 +0200
+++ mythplugins/mythflix/mythflix/mythflix.pro	2006-02-02 17:52:35.000000000 +0200
@@ -4,7 +4,7 @@
 TEMPLATE = lib
 CONFIG += plugin thread warn_on debug
 TARGET = mythflix
-target.path = $${PREFIX}/lib/mythtv/plugins
+target.path = $${PREFIX}/lib64/mythtv/plugins
 INSTALLS += target
 
 uifiles.path = $${PREFIX}/share/mythtv/themes/default
