diff -ur ./mythweb/includes/init.php /tmp/mythweb/includes/init.php
--- ./mythweb/includes/init.php	2005-02-28 00:52:48.000000000 +0200
+++ /tmp/mythweb/includes/init.php	2005-10-26 00:11:25.000000000 +0300
@@ -14,7 +14,7 @@
     require_once 'includes/errordisplay.php';
 
 // Load the user-defined configuration settings
-    require_once 'config/conf.php';
+    require_once '/etc/mythweb/conf.php';
 
 // Clean up some variables
     if (!ereg('/$', $_SERVER['DOCUMENT_ROOT']))
--- ./mythweb/config/conf.php~	2005-11-04 02:25:34.000000000 +0200
+++ ./mythweb/config/conf.php	2005-11-04 02:26:32.000000000 +0200
@@ -29,7 +29,7 @@
 //    defined as mythtv.mydomain.com in server_domain above, mail will go to
 //    mythweb_errors@mydomain.com.
 //
-    define('error_email', 'mythweb_errors@'.preg_replace('/.*?\b([\w\-]+\.[\w\-]+)$/', '$1', server_domain));
+    define('error_email', 'root@'.preg_replace('/.*?\b([\w\-]+\.[\w\-]+)$/', '$1', server_domain));
 
 // For the "movies" search -- set this to the word your listings provider uses to
 //   describe movies/films/peliculas/etc.
--- /mythweb/includes/mythbackend.php~	2005-10-22 13:31:12.000000000 +0300
+++ /mythweb/includes/mythbackend.php	2005-11-04 23:31:31.707600269 +0200
@@ -335,7 +335,7 @@
             // Make sure the local path exists
                 $path = '';
                 foreach (split('/+', dirname($pngpath)) as $dir) {
-                    $path .= $path ? '/' . $dir : $dir;
+					$path .= $path ? ('/' . $dir) : ($dir ? $dir : "/$dir");
                     if(!is_dir($path) && !mkdir($path, 0755))
                         trigger_error('Error creating path for '.$path.': Please check permissions.', FATAL);
                 }
--- ./mythweb/includes/init.php	2006-01-07 02:10:47.000000000 +0200
+++ /tmp/init.php	2006-01-08 01:02:40.000000000 +0200
@@ -232,12 +232,12 @@
     define('theme_url', root.theme_dir);
 
 // Make sure the data directory exists and is writable
-    if (!is_dir('data') && !mkdir('data', 0755)) {
+    if (!is_dir('/var/cache/mythweb') && !mkdir('/var/cache/mythweb', 0755)) {
         $Error = 'Error creating the data directory. Please check permissions.';
         require_once 'templates/_error.php';
         exit;
     }
-    if (!is_writable('data')) {
+    if (!is_writable('/var/cache/mythweb')) {
         $process_user = posix_getpwuid(posix_geteuid());
         $Error = 'data directory is not writable by '.$process_user['name'].'. Please check permissions.';
         require_once 'templates/_error.php';
@@ -245,7 +245,7 @@
     }
 
 // New hard-coded cache directory
-    define('cache_dir', 'data/cache');
+    define('cache_dir', '/var/cache/mythweb');
 
 // Make sure the image cache path exists and is writable
     if (!is_dir(cache_dir) && !mkdir(cache_dir, 0755)) {
