diff -dur -x'*~' mythplugins-0.22.orig/mythweb/mythweb.php mythplugins-0.22/mythweb/mythweb.php
--- mythplugins-0.22.orig/mythweb/mythweb.php	2009-01-20 07:42:03.000000000 +0100
+++ mythplugins-0.22/mythweb/mythweb.php	2010-01-23 14:34:35.000000000 +0100
@@ -12,6 +12,9 @@
  *
 /**/
 
+// MythWeb seems uses relative paths for everything... let it work
+  chdir('/usr/share/mythweb');
+
 // Add a custom include path?
     if (!empty($_SERVER['include_path']) && $_SERVER['include_path'] != '.')
         ini_set('include_path', $_SERVER['include_path'].PATH_SEPARATOR.ini_get('include_path'));
