diff -urNp -x '*.orig' mythplugins-0.26.1.org/mytharchive/mytharchivehelper/main.cpp mythplugins-0.26.1/mytharchive/mytharchivehelper/main.cpp
--- mythplugins-0.26.1.org/mytharchive/mytharchivehelper/main.cpp	2013-08-15 16:44:02.000000000 +0200
+++ mythplugins-0.26.1/mytharchive/mytharchivehelper/main.cpp	2021-09-26 12:20:08.236252907 +0200
@@ -103,6 +103,10 @@ namespace
     };
 }
 
+#ifndef INT64_C    // Used in ffmpeg headers to define some constants
+#define INT64_C(v)   (v ## LL)
+#endif
+
 class NativeArchive
 {
   public:
